models/audit.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+  var Audit = sequelize.define('Audit', {
+    table_name: DataTypes.STRING,
+    field_id: DataTypes.STRING,
```

Comment:
Same comments as for migration, probably need to regenerate the model to correct the names of properties.